README.md

```diff
@@ -75,3 +75,8 @@ Ideally, it should be run at least once every hour.
 
 The algorithm performs one Algolia search operation per collection updated.
 
+### Logger
+
+The script logs all the requests to Shopify in a  `shopify_api.log` file located in the `/log` folder. This file might get pretty big if the script is run often.
+If you want to stop logging the requests, comment the line `38` of the `bin/init.rb` file.
```

Comment:
I will recommend specifying the line which does the logging as numbers can change pretty easily with code changes.

Reply:
Agreed, done here : https://github.com/algolia/shopify-seo/pull/3/commits/b6d402884e6acc3c95dc69d3a0203994a537ba93